src/components/location/location.service.ts

```diff
@@ -259,7 +244,13 @@ export class LocationService {
               id,
             }),
           ],
-          ...this.property('name', input.name, 'newRegion'),
+          ...property(
+            'name',
+            input.name,
+            'newRegion',
+            'name',
+            'Property:LocationName'
```

Comment:
Do you need to remove `Property`? Or does Cypher not care if there's a duplicate label?

Reply:
Oh... I think I need to remove `Property`